meetup/tests.py

```diff
@@ -47,65 +91,62 @@ def test_meetup_link_is_unique(self):
     self.assertRaises(IntegrityError, n.save)
 
   def test_getting_all_associated_pizzas(self):
-    meetup= Meetup.objects.create(name="Meeetup1", meetup_link='http://meetup.com/some-meetup')
-    place = meetup.pizza_places.create(name="Pete Zazz")
-    self.assertEquals(place, meetup.pizza_places.all()[0])
+    meetup= Meetup.objects.create(name="Meetup1", meetup_link='http://meetup.com/some-meetup')
+    place = meetup.pizza_places.create(name="Pete Zazz", yelp_link='https://www.yelp.com/biz/prince-st-pizza-new-york')
```

Comment:
Another reason not to do work in your constructors: What if instantiating a Pizza Place with a `yelp_link` called out to the Yelp API to fetch its data? :scream_cat: That would be very hard to prevent in a test environment like this. 

(I know that's not what's actually happening here, but this line brought the example to mind.)